includes/polling/toner.inc.php

```diff
@@ -5,12 +5,9 @@
 foreach ($toner_data as $toner) {
     echo 'Checking toner '.$toner['toner_descr'].'... ';
 
-    if ($toner['toner_capacity_oid']) {
```

Comment:
I'm confused over why this would be removed. toner_capacity_oid shouldn't exist for you but we still need to use it for those that do.